src/test/scala/org/ergoplatform/explorer/db/dao/OutputsDaoSpec.scala

```diff
@@ -60,6 +69,20 @@ class OutputsDaoSpec extends FlatSpec with Matchers with BeforeAndAfterAll with
     val unspentSum = unspent.map{_.value}.sum
 
     dao.sumOfAllUnspentOutputsSince(0L).transact(xa).unsafeRunSync() shouldBe unspentSum
+
+    // Token seller contract from AssetsAtomicExchange
+    // http://github.com/ScorexFoundation/sigmastate-interpreter/blob/633efcfd47f2fa4aa240eee2f774cc033cc241a5/contract-verification/src/main/scala/sigmastate/verification/contract/AssetsAtomicExchange.scala#L34-L34
```

Comment:
this url will be broken, maybe better to copy the contract body here, since it is strictly connected to outputs.csv

Reply:
@aslesarenko Why do you think this URL will be broken? According to [github](https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files#press-kbdykbd-to-permalink-to-a-file-in-a-specific-commit) it's a permanent link to the file in that commit.